python/lsst/summit/utils/astrometry/fastStarTrackerAnalysis.py

```diff
@@ -50,6 +54,29 @@
     "plotSourcesOnImage",
 )
 
+KNOWN_CAMERAS = ("regular", "wide", "fast")
```

Comment:
Not narrow? Shouldn't this be KNOWN_CAMERA_TYPES then, if you plan to have another camera with `cameraType="regular"`?

Reply:
I don't think the `KNOWN_CAMERAS` needs renaming as they're physical cameras, not types, but yeah, this comment and all the others are absolutely right that I got part way through renaming `regular` to `narrow` but missed loads of places.